### PR TITLE
Feature/add logs

### PR DESCRIPTION
**Pull request by alxx88**

### CODE REVIEW

**Comment by kmih** [comment]
:+1:

File: src/main/java/com/epam/idea/rest/aspect/ServicesAspect.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    @After("execution(* com.epam.idea.core.service.impl..*(..))")
+    public void logAfter(JoinPoint joinPoint){
+        log.log(
+                String.format(logPatternAfter,
+                        (joinPoint != null && joinPoint.getThis() != null)?joinPoint.getThis().getClass().getName():null,
+                        (joinPoint != null && joinPoint.getSignature() != null)?joinPoint.getSignature().getName():null
+                )
+        );
+    }//logAfter()
```
Comment (Hanna-Aliakseichykava) [inline src/main/java/com/epam/idea/rest/aspect/ServicesAspect.java:41]: May be it makes sense to create methods to obtain class name and method name to decrease duplication.

**Comment by Hanna-Aliakseichykava** [comment]
:+1:

**Comment by LostFan** [comment]
:+1:

**Pull request merged**